src/Apps/Collect/Components/Base/CollectFilterContainer.tsx

```diff
@@ -0,0 +1,94 @@
+import { CollectFilterContainer_viewer } from "__generated__/CollectFilterContainer_viewer.graphql"
+import { ContextConsumer } from "Artsy"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Responsive } from "Utils/Responsive"
+import { CollectRefetchContainer } from "./CollectRefetch"
+
+import { FilterContainer } from "../Filters"
+
+export interface CollectFilterContainerProps {
+  viewer: CollectFilterContainer_viewer
+}
+export class CollectFilterContainer extends Component<
+  CollectFilterContainerProps
+> {
+  render() {
+    const { filter_artworks } = this.props.viewer
+    const { aggregations } = filter_artworks
+    const mediumAggregation = aggregations.find(agg => agg.slice === "MEDIUM")
+
+    return (
+      <ContextConsumer>
+        {({ user, mediator }) => {
+          return (
+            <Responsive>
+              {({ xs }) => {
+                return (
+                  <FilterContainer
+                    isMobile={xs}
+                    user={user}
+                    mediator={mediator}
+                    mediums={mediumAggregation.counts as any}
+                  >
+                    {({ filters }) => (
+                      <CollectRefetchContainer
+                        viewer={this.props.viewer}
+                        filters={filters}
+                      />
+                    )}
+                  </FilterContainer>
+                )
+              }}
+            </Responsive>
+          )
+        }}
+      </ContextConsumer>
+    )
+  }
+}
+
+export const CollectFilterFragmentContainer = createFragmentContainer(
+  CollectFilterContainer,
```

Comment:
Not sure how necessary this is in this instance, but render props code can be made more useful throughout a full class if passed in and accessed as props (similar to whats [happening here](https://github.com/artsy/reaction/blob/912e401b1c148fdb7c8c596d9d2f863da6c73b18/src/Apps/Comparables/Filter.tsx#L68-L88). Since the component is already been wrapped it makes this pattern easy to implement.